CHANGELOG.md

```diff
@@ -3,7 +3,17 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 2.4.0 - 2023-09-25
+## 2.4.1dev - unreleased
+
+### `Added`
+
+### `Changed`
+
+### `Fixed`
+
+### `Dependencies`
+
+## 2.4.0 - 2023-09-26
```

Comment:
25 -> 26?

Reply:
The release came out today (10m ago) rather than yesterday but ofc I forgot to update before merging :sweat_smile: